packages/campaignclassic/android/src/main/java/com/adobe/marketing/mobile/reactnative/optimize/RCTAEPCampaignClassic.java

```diff
@@ -0,0 +1,72 @@
+/*
+ Copyright 2022 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or
+ agreed to in writing, software distributed under the License is distributed on
+ an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, either
+ express or implied. See the License for the specific language governing
+ permissions and limitations under the License.
+ */
+package com.adobe.marketing.mobile.reactnative.campaignclassic;
```

Comment:
Is the name of the folder wrong? Should it be `java/com/adobe/marketing/mobile/reactnative/campaignclassic`?

Reply:
Totally is! Sorry vscode cut off the folder name 😄